src/api/Cargo.toml

```diff
@@ -10,7 +10,8 @@ common-base = { path = "../common/base" }
 common-error = { path = "../common/error" }
 common-time = { path = "../common/time" }
 datatypes = { path = "../datatypes" }
-greptime-proto = { git = "https://github.com/GreptimeTeam/greptime-proto.git", rev = "6bfb02057c40da0e397c0cb4f6b87bd769669d50" }
+# greptime-proto = { git = "https://github.com/GreptimeTeam/greptime-proto.git", rev = "6bfb02057c40da0e397c0cb4f6b87bd769669d50" }
+greptime-proto = { git = "https://github.com/lizhemingi/greptime-proto.git", rev = "21bff071f36568516a7738f502c80bcd39e3102d" }
```

Comment:
Related pr: https://github.com/GreptimeTeam/greptime-proto/pull/38

Reply:
Already merged, please use the official repo.